src/Compilers/VisualBasic/Portable/Symbols/Symbol.vb

```diff
@@ -62,6 +62,16 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Get
         End Property
 
+        ''' <summary>
+        ''' Gets the token for this symbol as it appears in metadata. Most of the time this Is 0,
+        ''' as it Is when the symbol Is Not loaded from metadata.
+        ''' </summary>
+        Public Overridable ReadOnly Property MetadataToken As Integer Implements ISymbol.MetadataToken
+            Get
+                Return 0
```

Comment:
Similar comment as for C#. It seems like it would be less error-prone to make the property abstract and implement it to return 0 on all direct subclasses of `Symbol`.